conans/test/functional/command/inspect_test.py

```diff
@@ -397,7 +397,10 @@ def test_options_dict(self):
         client = TestClient()
         client.save({"conanfile.py": self.conanfile})
         client.run("inspect . --raw=options")
-        self.assertEqual("{'foo': [True, False], 'bar': [True, False]}", client.out)
+        # The output is a dictionary, no order guaranteed, we need to compare as dict
+        import ast
```

Comment:
Why not just ``eval()``?

Reply:
It could be changed by `eval`, but we are already using `ast` in Conan and `literal_eval` is safer. Ok, it is the output of a test, should be safe, but... why not?